components/toolbar/appearance.md

```diff
@@ -12,6 +12,41 @@ position: 35
 
 This article outlines the available ToolBar parameters, which control its appearance.
 
+## FillMode
+
+The `FillMode` parameter controls if the ToolBar will have a background and borders. To set the parameter value, use the `string` members of the static class `ThemeConstants.ToolBar.FillMode`.
+
+| `FillMode` Class Member | String Value |
+| --- | --- |
+| `Flat` | `"flat"` |
+| `Solid` (default) | `"solid"` |
```

Comment:
Is there any reason the `default (Solid)` is positioned second in the list of values? I would recommend positioning it first. The same as the other components articles that includes such a section.